packages/main/src/plugin/recommendations/recommendations-registry.spec.ts

```diff
@@ -16,23 +16,49 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 
-import { beforeEach, expect, test, vi } from 'vitest';
+import { beforeEach, describe, expect, test, vi } from 'vitest';
 
 import type { ConfigurationRegistry } from '/@/plugin/configuration-registry.js';
+import type { Featured } from '/@/plugin/featured/featured.js';
+import type { FeaturedExtension } from '/@/plugin/featured/featured-api.js';
 
 import { RecommendationsRegistry } from './recommendations-registry.js';
 
 let recommendationsRegistry: RecommendationsRegistry;
 
-const registerConfigurationsMock = vi.fn();
+const mocks = vi.hoisted(() => ({
+  registerConfigurationsMock: vi.fn(),
+  getRecommendationIgnored: vi.fn(),
+}));
```

Comment:
why do we need to use that pattern

we already has `const registerConfigurationsMock = vi.fn();`

Reply:
AFAIK we don't need these hoisted

Reply:
I removed them !